.kodiak.toml

```
@@ -0,0 +1,53 @@
+# .kodiak.toml
+version = 1
+
+[merge]
+
+require_automerge_label = false
+
+# if this title regex matches, Kodiak will not merge the PR. this is useful
+# to prevent merging work in progress PRs
+blacklist_title_regex = "^WIP.*" # default: "^WIP.*", options: "" (disables regex), a regex string (e.g. ".*DONT\s*MERGE.*")
+
+# if these labels are set Kodiak will not merge the PR
+blacklist_labels = ["wip"] # default: [], options: list of label names (e.g. ["wip"])
+
+# choose a merge method. If the configured merge method is disabled for a
+# repository, Kodiak will report an error in a status message.
+method = "squash" # default: "merge", options: "merge", "squash", "rebase"
+
+# once a PR is merged into master, delete the branch
+delete_branch_on_merge = true # default: false
+
+# if you request review from a user, don't merge until that user provides a
+# review, even if the PR is passing all checks
+block_on_reviews_requested = true # default: false
+
+# if there are running status checks on a PR when it's up for merge, don't
+# wait for those to finish before updating the branch
+optimistic_updates = true # default: true
```

We should definitely wait for checks to pass before merging @nglgzz 

```suggestion
optimistic_updates = false # default: true
```